ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -217,7 +233,11 @@ private Optional<SafeFuture<BlockImportResult>> handleKnownBlock(final SignedBea
 
   private SafeFuture<BlockImportResult> handleBlockImport(
       final SignedBeaconBlock block,
+      final Optional<BlobsSidecar> blobsSidecar,
       final Optional<BlockImportPerformance> blockImportPerformance) {
+    if (blobsSidecar.isPresent()) {
+      LOG.trace("blobsSidecar is present");
```

Comment:
I'd log something about it here, not very big but with some info. If we start logging...

Reply:
Yeah was just a way to use the variable and avoid the compilation issue of unused variable